src/Ushahidi/Modules/V5/Actions/Role/Commands/CreateRoleCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Actions\Role\Commands;
+
+use App\Bus\Command\Command;
+
+class CreateRoleCommand implements Command
+{
+    /**
+     * @var array
+     */
+    private $input;
```

Comment:
We should specify fields inside the input within the command. Array does not give us clarify of what to expect inside the input and makes it more difficult to validate.

Reply:
I agree, What do you suggest? Using separate properties for fields?

Reply:
I'm thinking of using the current Entity as DTO for the Input

Reply:
CreateRoleCommand is a DTO use separate properties for fields. We don't want stuff we cannot easily validate.